### ISSUE TITLE
Issue installing Windows app

### ISSUE THREAD

**Issue by krosenewhope** [gruepr/gruepr#6]
We attempted to install Gruepr for the first time and encountered this error message:

![image](https://github.com/user-attachments/assets/55417825-31b9-4b51-b457-f7099c43725e)

We tried the install on Windows 10 and 11, granted the appropriate admin permissions, and also tried it on a PC with full admin rights. The install package doesn't place the shortcuts on the desktop or in the Start Menu, and it errors that it can't find a .dll file. Has this been reported by anyone else? Our PCs are on a network, but it also didn't work on a personal PC with no restrictions.

**Comment by joshuahertz** [comment]
This is the first report of an error like this. I will look into it soon. In the meantime, the error message suggests the computers are missing 1 or 2 common used files from Microsoft. Installing them should fix this; official MS source here: https://www.microsoft.com/en-ca/download/details.aspx?id=48145After downloading and installing both of these files, restart the computer and retry the gruepr installation. On Aug 22, 2024, at 4:25 PM, krosenewhope ***@***.***> wrote:﻿
We attempted to install Gruepr for the first time and encountered this error message:
image.png (view on web)
We tried the install on Windows 10 and 11, granted the appropriate admin permissions, and also tried it on a PC with full admin rights. The install package doesn't place the shortcuts on the desktop or in the Start Menu, and it errors that it can't find a .dll file. Has this been reported by anyone else? Our PCs are on a network, but it also didn't work on a personal PC with no restrictions.
